homeassistant/bootstrap.py

```diff
@@ -123,9 +124,13 @@ def async_from_config_dict(config: Dict[str, Any],
         new_config[key] = value or {}
     config = new_config
 
+    hass.config_entries = config_entries.ConfigEntries(hass, config)
+    yield from hass.config_entries.async_load()
```

Comment:
I think there is no better solution at the moment. But we should add `config_entries = None` to `core.HomeAssistant` so we know there is a config_entries they will comes later.

Reply:
I think that we shouldn't.  Right now we have these layers:

Core = the Home Assistant core. Unaware of components or config.
Config / config entries = the configuration layer
Entity = Entity layer to manage entities

(I will update the architecture docs soon)